### PR TITLE
Add section about IANA Thing Model content type registration

### PR DESCRIPTION
**Pull request by sebastiankb**
solves #931

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1476.html" title="Last updated on May 12, 2022, 7:40 AM UTC (76398ed)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1476/a76e087...76398ed.html" title="Last updated on May 12, 2022, 7:40 AM UTC (76398ed)">Diff</a>

### CODE REVIEW

**Comment by sebastiankb** [comment]
@plehegar if this looks ok for you I would start to submit the official registration request at IANA.

**Review by plehegar — APPROVED** [review]

File: index.html
Comment (mmccool) [inline index.html:7325]: JSON Pointer is [RFC6901](https://w3c.github.io/wot-thing-description/#bib-rfc6901)].
RFC 6839 is just general guidance on how tags work but says that "if there is an extension the fragment identifier can be defined" but the parent is otherwise inherited, but JSON itself (the parent) does not define a fragment identifier.

I think we should *explicitly* state that JSON Pointer (the RFC above) is the default fragment identifier for TMs (AND TDs...)

File: index.html
Comment (mmccool) [inline index.html:7296]: ok, this looks good, I don't know why I saw the old text when I looked last

File: index.html
Comment (mmccool) [inline index.html:7333]: ideally we would say .tm.jsonld is preferred

**Comment by mmccool** [comment]
Never mind my above comments about security, it seems I was looking at an old version somehow.  I did a real review and it looks good, *except* I think we should make JSON Pointer the official fragment identifier for both TD and TM (otherwise it inherits from JSON, which does not in fact have one, so then TD and TM would not have one)

**Review by mmccool — CHANGES_REQUESTED** [review]

**Comment by sebastiankb** [comment]
from today's TD call:
- @sebastiankb will include @mmccool input
- after getting approval from @mmccool I'm going to merge

File: index.html
Comment (sebastiankb) [reply index.html]: done

File: index.html
Comment (sebastiankb) [reply index.html]: done

**Review by mmccool — APPROVED** [review]

**Pull request merged**
